### ISSUE TITLE
Get XML format command to preserve whitespace as XML value

### ISSUE THREAD

**Issue by hmobius** [redhat-developer/vscode-xml#96]
I frequently want to space-separate a value in my XSL output and do so by using the following XSL

```
    <xsl:template match="d:blah">
        <xsl:text> </xsl:text>
        <xsl:element name="{local-name()}">
            <xsl:apply-templates />
        </xsl:element>
    </xsl:template>
```

But when I format the XSL using your extension, it removes the space from between the <xsl:text> tags. Please can you add a setting to preserve whitespace?

**Comment by NikolasKomonen** [comment]
For sure. This is an upstream issue, it can be found here : https://github.com/angelozerr/lsp4xml/issues/273

**Comment by NikolasKomonen** [comment]
@hmobius this will be available in the v0.4.0 release
